src/foam/u2/tag/Select.js

```diff
@@ -45,8 +47,15 @@ foam.CLASS({
       this.setChildren(this.slot(function(choices, placeholder) {
         var cs = [];
 
-        if ( placeholder ) {
-          cs.push(self.E('option').attrs({disabled: 'disabled'}).add(self.placeholder));
+          if ( placeholder ) {
+            if (this.optional) {
+              cs.push(self.E('option').attrs({
+                selected: self.data == -1 ? true : undefined,
+                value: -1
+            }).add(self.placeholder));
+          } else {
```

Comment:
Your indentation makes it look like the else is for the first if statement, but it's actually for the inner one.